modules/special-rules/pages/super-lifta-droppa.adoc

```diff
@@ -3,18 +3,36 @@
 
-Ork Mekboys use the weapon to focus a traktor beam on the target and then lift the target high above the ground -- or above a juicy secondary target nearby.
-Then they drop the first target onto the secondary target...
+Ork Mekboys use the weapon to focus a traktor beam on the target and then lift the target high above the ground -- or above juicy secondary targets nearby.
+Then they drop the first target onto the secondary targets...
 
 ---
 
-When you shoot with a {super-lifta-droppa}:
+When you make a {super-lifta-droppa} shot:
 
-. You can target Vehicle and War Engine units, but not Infantry units.
-. To hit and lift the target unit, roll a D6: You need to score higher than the initial Damage Capacity value of the target unit.
-(Vehicle units have a Damage Capacity value of 1; War Engine units vary, so check their data sheet.)
-. For each point by which your score is higher than the target unit's initial Damage Capacity value, you can move it up to 1 cm in any direction to determine where the {super-lifta-droppa} will drop it.
-. The target unit is dropped at the spot you chose, and is automatically destroyed -- if it is a War Engine unit, roll on its Catastrophic Damage table immediately.
-. If there is a secondary target at the spot where the target unit is dropped, roll a D6: On a result of 4+ the secondary target takes a hit.
+. Pick any Vehicle or War Engine target within range and line of sight, even a specific unit in a detachment.
+You cannot target Infantry units.
+. Roll a D6.
+If you roll a result that is higher than the initial Damage Capacity value of the target unit, then the weapon hits.
++
+Vehicle units usually have a Damage Capacity value of 1.
+War Engine units vary, so check the target unit's data sheet.
+. If the target unit has the {save} ability or the {holo-fields} ability, then your opponent may attempt the save.
+If they are successful then the target is unharmed and nothing further happens -- do not continue below.
+. For each point by which your result is higher than the target unit's initial Damage Capacity value, you can move the unit up to 1 cm in any direction, above other game pieces.
+This is where the {super-lifta-droppa} will drop the target unit.
+. Resolve the drop:
+* The target unit is automatically destroyed.
+If it is a War Engine unit, roll for a result on its Catastrophic Damage table.
+* For each unit that is under the target unit, roll a D6.
+On a roll of 4+, that unit suffers a hit.
++
+{blank}[TODO: (a) How do we define 'under', should we reference some of the same points as for the Barrage template; (b) How do we resolve differences between based vehicles and non-based vehicles... or can only War Engine units be dropped on more than one target? The latter seems best to me (in which case we should perhaps revert the plurals at the start of this topic to singular) -- but either way we should define it I think.]
```

Comment:
Suggestion added so you can see what I'm referring to...

```suggestion
```

(a) We could do. Chris and I have just been playing it as-is, so anything underneath in any way is rolled for. I think anything else is going to be hard to write without having to cover lots of edge-cases. Maybe leave it as 'under the target unit' or 'under the dropped unit', and wait to see if people start asking how it works.

(b) It's not been added into the Main rules section yet, but as per FB discussions about Barrage, measuring ranges and base contact etc, based vehicles will use the base for everything, and non-based vehicles will treat their main hull or body as 'the base'. So I don't think there's a distinction to make there.

Why would war engines be the only thing that can be dropped on more than one target? The only place the original text makes a distinction between unit types for any part of the Super Lifta-Droppa is that you can't pick up infantry units. That's not a criticism, I don't know where the distinction is coming from!

Reply:
(a) Yeah, I haven't edited the Barrage template targetting bits yet, but I figured it might be a quick win to reuse the same sense here. I agree that this would be simplest and avoid a variation that invites a whole other lot of questions about edge cases.
I must say though, my mind immediately went "what counts as 'under'...?" and it seems like a question that any player will have to ask themselves when they need to apply this rule. We can leave it for now, but I think we should define it at some point. 

(b) My problem with this is that it is inherently 'unfair' for players who don't base their vehicles, and incentivises big bases. Bigger base = bigger area to hit more units below. Me no likey. This is the big reason why I think Vehicle units should only strike a single unit, it does away with this problem entirely -- hence the distinction.

War engines are giant, and so we accept or expect that they might hit multiple units below when dropped -- and this gives some payoff for the greater difficulty to pick them up in the first place. There will still be some 'unfair' variation in base/miniature sizes here, but much less of an issue overall than with vehicles.

Reply:
> (b) My problem with this is that it is inherently 'unfair' for players who don't base their vehicles, and incentivises big bases. Bigger base = bigger area to hit more units below. Me no likey. This is the big reason why I think Vehicle units should only strike a single unit, it does away with this problem entirely -- hence the distinction.

Do you mean 'unfair for players who _do_ base their vehicles'? Because those are the players who would suffer more from Lifta-Droppas. 

As someone who bases his vehicle models and would suffer more, it doesn't bother me. It's part-and-parcel of the trade-off of basing your vehicle models - you can't bunch them up as tightly, they're easier to hit with templates for the same model density, they have bigger Snap-fire areas, they have a wider 'spread' of range in different directions. It all averages out in the end, even if there are specific situations where it's better or worse than no base.

The flip-side is that my suspension of disbelief balks at the idea that a dropped Ork Buggy, Sentinel or War Walker has exactly the same 'impact area' as a Land Raider or Stompa.

I'm also concerned that there are balance issues as well as being a rules change, as only being able to hit a single secondary target is a _substantial_ reduction in the power level of Lifta-Droppas. And Lifta-Droppas are already a points cost issue for Gargants due to costing the same as 5 Firepower, but having the same Range as a FP8 battery.

> War engines are giant, and so we accept or expect that they might hit multiple units below when dropped -- and this gives some payoff for the greater difficulty to pick them up in the first place. There will still be some 'unfair' variation in base/miniature sizes here, but much less of an issue overall than with vehicles.

It's not going to be less of an issue. 😕 

Slaaneshi Scout Titans came with 25 mm slottabases. AT18 Warhounds come with 80 mm bases which is more than ten times the area. Even ignoring the new models I've seen the old ones on anything from 40 mm square to 60 mm round.

All that said, how about a compromise? 

The dropped unit can't land on more than 2x its initial DC. So standard vehicles can land on two, DC2 Land Raiders, Stompas, Squiggoths etc. can land on four (but can only be moved 4 cm), and DC4 war engines can potentially land on eight, but can only be moved 2 cm so are almost never going to reach eight units.

Reply:
Not that it's entirely relevant, given the other differences between the editions, but in SM2/NetEpic the Lifta-Droppa:

1. Used a barrage template with Range 75 cm.
2. Hit everything under the template on a 4+, whether they were infantry, vehicles, or super-heavies like Baneblades.
3. Applied a -4 save mod, anything failing was picked up (and would be destroyed).
4. Picked up units were moved up to 6 cm (not depending on dice rolls), then scattered 2d6 cm and dropped.
5. The area of effect of the drop was the size of the dropped model.
6. Anything dropped on auto-died unless its save was better than the dropped model.

But anything with a location chart (so Titans) couldn't be picked up.

They were massively destructive weapons compared to the E40k version, but that applies to a lot of stuff!

Anyway, I think this could be another area where it would be useful to find out how people have _actually_ been playing it!

Reply:
>Do you mean 'unfair for players who do base their vehicles'? Because those are the players who would suffer more from Lifta-Droppas.

Yes, sorry, that's what I meant. Normally you don't use a players own 'area choices' against their own forces, and my mind wandered :-D

>It's part-and-parcel of the trade-off of basing your vehicle models... It all averages out in the end...

Yes, I suppose that's true. I was thinking that 'one unit only' is an easy way to even it out, but I take your point that it'd make the Super Lifta-droppa too weak. (I haven't analysed it closely next to Death Ray or whatever to call that one either way, so I take your word for it.)

>The flip-side is that my suspension of disbelief balks at the idea that a dropped Ork Buggy, Sentinel or War Walker has exactly the same 'impact area' as a Land Raider or Stompa.

If anything, I see it sort of the other way around -- a Land Raider is bigger than those others, yes, but not so much bigger that it is likely to straddle average gap that is likely to exist between units on the battlefield. It's just more likely to crush the one thing that it (might) land on. But I'm not wedded to it either way.

>>War engines are giant, and so we accept or expect that they might hit multiple units below when dropped -- and this gives some payoff for the greater difficulty to pick them up in the first place. There will still be some 'unfair' variation in base/miniature sizes here, but much less of an issue overall than with vehicles.

>It's not going to be less of an issue. 😕

>Slaaneshi Scout Titans came with 25 mm slottabases. AT18 Warhounds come with 80 mm bases which is more than ten times the area. Even ignoring the new models I've seen the old ones on anything from 40 mm square to 60 mm round.

That's true. I was just thinking in terms of there being fewer opportunities to target war engines in the first place, and less chance to pick them up, hence less of a problem. But the crazy variation in base size these days is an issue.

>All that said, how about a compromise?
>
>The dropped unit can't land on more than 2x its initial DC. So standard vehicles can land on two, DC2 Land Raiders, Stompas, Squiggoths etc. can land on four (but can only be moved 4 cm), and DC4 war engines can potentially land on eight, but can only be moved 2 cm so are almost never going to reach eight units.

I don't know — I'm not against it in principle, but sounds like more trouble than it is worth, especially given the small distance for war engines, as you say.

We can leave the whole idea alone if you like, it was just a thought.

Reply:
>They were massively destructive weapons compared to the E40k version, but that applies to a lot of stuff!

True dat!

Reply:
The FB discussions only had a couple of responses, but they were very firmly of the opinion that you roll to hit everything under the dropped unit, no matter how much was covered.

https://www.facebook.com/groups/1952676958177757/posts/3836846379760796/

Reply:
Cool. As I say, (b) was just a thought that popped up in relation to the main query (a), and partly driven by what I erroneously thought was a change in rule because of the switch from singular to plural at the start of the topic. So, nothing to resolve for (b) I think?

For my primary query (a), am I right to think that we should use the **exact** same determinations for 'under' as we do for the Barrage template?

Reply:
> For my primary query (a), am I right to think that we should use the **exact** same determinations for 'under' as we do for the Barrage template?

No! 

> you roll to hit everything under the dropped unit, **no matter how much was covered**.

So if any part of a unit is under any part of the dropped unit, you roll to hit. In terms of comparing to a Barrage template, you have to think in terms of everything being a vehicle unit.

Reply:
Okay, that's the main thing I wanted to know -- we have a different definition of 'under' that we need to spell out. Or at least, variation in the clauses for an existing definition, to suit this context.

I'll have a think about that alongside the other edits when I next get a chance to look at it. Time has gone into our conversations rather than actual editing this evening, but this has all been necessary to help with the edits.

Reply:
Okay, we could add something to the effect of:

>NOTE: In this case, we define 'under' as any part of the unit’s base -- no matter the unit type -- is under the dropped unit or touched by the edge of the outline of the dropped unit.

...but I really don't like the way that this breaks with the conditions we state at `barrage-templates/adoc`.
I get that the weapon might be seen as a bit weak, but I don't think that we should try to offset this by making infantry arbitrarily easier to hit than is usual for the otherwise similar template rules, just for this special case.
Such an odd break in the continuity feels like bad practice and needlessly complex to me.

Reply:
Damn - this conversation went outdated when I adjusted the TODO note. I'll repeat my last message on the current iteration rather than just leave it buried here.